lib/pure/collections/sequtils.nim

```diff
@@ -1106,16 +1122,12 @@ when isMainModule:
     doAssert newSeq[int](0).mapIt(it) == @[]
 
   block: # mapIt redifinition check, see https://github.com/nim-lang/Nim/issues/8580
-    let t = [1,2].mapIt(it)
-    doAssert t == @[1,2]
+    let s2 = [1,2].mapIt(it)
```

Comment:
`s2` is correct here as test case for https://github.com/nim-lang/Nim/issues/8580 since implementation of `mapIt` now uses s2, not t.